pkg/pb/referralmanager.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+option go_package = "pb";
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "metainfo.proto";
+import "orders.proto";
+
+package referralmanager;
+
+// ReferralManager is a service for handling referrals
+service ReferralManager {
+  // GetTokens retrieves a list of unredeemed tokens for a user
+  rpc GetTokens(GetTokensRequest) returns (GetTokensResponse);
+  // ValidateToken validates a referral token from referral manager
+  rpc ValidateToken(ValidateTokenRequest) returns (ValidateTokenResponse);
+  // SaveUserInfo saves newly created user info in referral manager
+  rpc SaveUserInfo(SaveUserInfoRequest) returns (SaveUserInfoResponse);
+}
+
+message GetTokensRequest{
+    bytes user_id = 1;
+    bytes node_id = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
+}
+
+message GetTokensResponse {
+ repeated bytes token = 1;
+}
+
+message ValidateTokenRequest {
+    bytes token = 2;
+    bytes node_id = 3;
```

Comment:
what does `node_id` for a validate token request mean? Is it the "redeeming" satellite ID or the "origin" satellite ID? Also, same comment as above: I think `satellite` should be in the field name.